fizzbuzz/FizzBuzzTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+ /*
+  * Print the numbers 1..100
+  * For multiples of 3, print "Fizz" instead of the number
+  * For multiples of 5, print "Buzz" instead of the number
+  * For multiples of 3 and 5, print "FizzBuzz" instead of the number
+  * Else it returns the number and a new line
+  */
+
+require_once __DIR__ . '/FizzBuzz.php';
+
+class TestFizzBuzz extends PHPUnit_Framework_TestCase
+{ 
+    private $fizzBuzz;
+
+    public function setUp()
+    {
+        $this->fizzBuzz = new FizzBuzz([
+            new MultipleKnowledge(3, 'Fizz'),
+            new MultipleKnowledge(5, 'Buzz'),
+            //new MultipleKnowledge(7, 'Bang'),
+        ]);
+    }
+
+    public function test0IsAMutipleOf3And5()
+    {
+        $this->assertEquals(
```

Comment:
I would test the multiple cases at the domain level, calling `$this->fizzBuzz->play()`. You can then test PHP_EOL presence with tests focused on the UI